src/System.Data.Common/tests/System/Data/Common/DataColumnMappingTest.cs

```diff
@@ -0,0 +1,70 @@
+using System.Data.Common;
+using Xunit;
+
+namespace System.Data.Tests.Common
+{
+    public class DataColumnMappingTest
+    {
+        [Fact]
+        public void DataSetColumn_Get_WhenNotSet()
+        {
+            DataColumnMapping dataColumnMapping = new DataColumnMapping();
+
+            string dataSetColumn = dataColumnMapping.DataSetColumn;
+
+            Assert.Equal("", dataSetColumn);
```

Comment:
I tend to use string.Empty explicitly where strings aren't concatenated so like in this example `Assert.Equal(string.Empty, dataSetColumn)`. I don't expect the author to change all places, it was more a suggestion than a change request.